kcidb/test_db.py

```diff
@@ -18,6 +19,17 @@ def test_schemas_main():
     assert_executes("", *argv,
                     stdout_re=r"4\.0: 4\.0\n4\.1: 4\.1\n")
 
+def test_reset(clean_database):
+    """Check Client.reset() works"""
+
```

Comment:
Could you remove this empty line? We'll be fine without it and the code will be shorter.